routes/api.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use App\Http\Controllers\GenreController;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+
+Route::get('/user', function (Request $request) {
+    return $request->user();
+})->middleware('auth:sanctum');
+
+
+//Routes for genres
```

Comment:
@damir-tahirovic Only authenticated librarians should be able to use these routes